README.md

```diff
@@ -28,7 +28,7 @@ See the [upgrade notes](https://github.com/greena13/react-hotkeys/releases/tag/v
 - [Optimized for larger applications](#Optimizations), with many hot keys active at once
 - Depends only on `prop-types` and a peer dependency of `react`
 - Uses rollup and Uglify and strips out comments and logging for a small production build
-- More than [1800 automated tests](https://github.com/greena13/react-hotkeys/tree/master/test)
+- More than [2000 automated tests](https://github.com/greena13/react-hotkeys/tree/master/test)
```

Comment:
🐥Splitting out a number of (large) test files into the intended behaviour when submatches are enabled and when they are not, significantly increased the test suite size.